app/javascript/packs/goodserviceApp/components/landingPage.jsx

```diff
@@ -241,9 +249,43 @@ class LandingPage extends React.Component {
     if (TEST_DATA) {
       this.setState({ trains: sampleData.routes, lines: sampleData.lines, loading: false });
     } else {
+      let params = qs.parse(location.search);
+      let use_median = params.median && params.median === 'true';
```

Comment:
Just tiny nitpick, let's keep variable names camelCase.

Reply:
Also, don't think we need the first half of the evaluation if we are evaluating with `===`?